Cartfile

```diff
@@ -1,5 +1,5 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.7
-github "mapbox/MapboxDirections.swift" ~> 0.15
+github "mapbox/MapboxDirections.swift" ~> 0.15.1
```

Comment:
I think this can remain actually. The only change that should occur is in Cartfile.resolved.